components/bio-formats-plugins/src/loci/plugins/in/importer-options.txt

```diff
@@ -189,9 +189,19 @@ default = false
 type = boolean
 label = Display_ROIs
 info = <b>Display ROIs</b> - \
-  Adds any ROIs in the file to ImageJ's ROI manager.
+  Adds any ROIs in the file to ImageJ.
 default = false
 
+[roiMode]
+label = ROIs_Import Mode:
+info = <b>ROI Import Mode</b> - \
+  ROIs can be imported to the ROI manager or an overlay.
+type = string
+default = ROI Manager
```

Comment:
On top of the `equalsIgnoreCase` fix added in https://github.com/openmicroscopy/bioformats/pull/2333/commits/0e45bd4b44e61dd57404a35418b496a157159e88, is it worth using the same nomenclature as `ROIS_MODE_MANAGER` for `default` and `values` i.e. `ROI manager` with lower case `m`?